### ISSUE TITLE
Date ISO8601 can be interpreted in two ways

### ISSUE THREAD

**Issue by jasollien** [BuildingSMART/BCF-API#165]
Look at these two dates:
2018-04-16T12:27:17.508+0000
2018-04-16T12:27:17.508+00:00

They are both compliant with ISO8601 since the ":" is optional.
Solibri uses ":", while bimsync is not using it.

Could we standardize this, or state that both values are valid?

Please comment on this, (including which format you are currently using), so we are alle prepared to discuss this on the next telecon.

**Comment by GeorgDangl** [comment]
In .NET, they're both treated equal (both for `DateTime.Parse()` and with Newtonsoft.Json). If ISO8601 treats them as equal, we should, too=)

```csharp
[Fact]
public void DatesAreTheSame()
{
    var withColon = "\"2018-04-16T12:27:17.508+0435\"";
    var withoutColon = "\"2018-04-16T12:27:17.508+04:35\"";

    var dateWithColon = JsonConvert.DeserializeObject<DateTimeOffset>(withColon);
    var dateWithoutColon = JsonConvert.DeserializeObject<DateTimeOffset>(withoutColon);

    Assert.Equal(dateWithColon, dateWithoutColon);
    Assert.Equal(dateWithColon.Offset, dateWithoutColon.Offset);
    Assert.Equal(dateWithoutColon.Offset, TimeSpan.FromMinutes(4 * 60 + 35));
}
```

**Comment by GeorgDangl** [comment]
Fixed in f4e5cfca1dff59916f8e521667a90464f75f0407
